docs/testbed/README.testbed.VsSetup.md

```diff
@@ -138,6 +139,17 @@ mkdir -p ~/veos-vm/images
 mv sonic-vs.img ~/veos-vm/images
 ```
 
+### Option 2: Use sonic-vpp image
+1. Follow the instructions from [sonic-platform-vpp](https://github.com/sonic-net/sonic-platform-vpp?tab=readme-ov-file#building-a-kvm-vm-image) and build a **kvm** vm image.
+2. Unzip the image, rename the image and copy it into `~/sonic-vm/images/` and also `~/veos-vm/images`
+```
+gzip -d sonic-vpp.img.gz
+mv sonic-vpp.img sonic-vs.img
+mkdir -p ~/sonic-vm/images
+cp sonic-vs.img ~/sonic-vm/images
+mkdir -p ~/veos-vm/images
+mv sonic-vs.img ~/veos-vm/images
+```
```

Comment:
Are these steps essentially the same as the above? If so, we can probably merge them.

Reply:
> Are these steps essentially the same as the above? If so, we can probably merge them.

It's slightly different. Was trying to avoid confusion for those who set up vtestbed for the first time.

Reply:
Addressed